frontend/src/pages/org/collections-list.ts

```diff
@@ -604,11 +611,11 @@ export class CollectionsList extends LiteElement {
             class="px-6 py-[0.6rem] flex gap-2 items-center whitespace-nowrap hover:bg-neutral-100"
             download
             @click=${(e: MouseEvent) => {
-              (
-                (e.target as HTMLAnchorElement).closest(
-                  "btrix-overflow-dropdown"
-                ) as OverflowDropdown
-              )?.hide();
+              const dropdown = (e.target as HTMLAnchorElement).closest(
+                "btrix-overflow-dropdown"
+              ) as OverflowDropdown;
```

Comment:
@emma-sg when I run `yarn lint`, it removes `as OverflowDropdown`. I'm unclear on which rule is causing this?

Reply:
Ah, I think the more correct way to do this is to supply `OverflowDropdown` as a generic type for `Element.closest`:
```ts
(e.target as HTMLAnchorElement)
  .closest<OverflowDropdown>("btrix-overflow-dropdown")
  ?.hide();
```

Reply:
I can see how this is confusing though, hmm, lemme poke around a little.